src/core/reducers/addons.js

```diff
@@ -17,6 +17,7 @@ export function denormalizeAddon(apiAddon) {
       ...apiAddon,
       // Set iconUrl to be consistent between disco and amo.
       iconUrl: apiAddon.icon_url,
+      is_restart_required: apiAddon.is_restart_required || false,
```

Comment:
Well, even though it's not necessary (since it will only be false for themes or extensions without files) I see that you were using this code to initialize it to `false`. My main confusion here is that it looks like `is_restart_required` is being accessed on the API object which looks like a typo since the API doesn't return it.

First, I think renaming it to `isRestartRequired` will help.

Second, how about initializing it more directly, like this?

```diff
diff --git a/src/core/reducers/addons.js b/src/core/reducers/addons.js
index 1b051c34..1a935e2b 100644
--- a/src/core/reducers/addons.js
+++ b/src/core/reducers/addons.js
@@ -28,7 +28,10 @@ export default function addon(state = initialState, action) {
   if (payload && payload.entities && payload.entities.addons) {
     const newState = { ...state };
     Object.keys(payload.entities.addons).forEach((key) => {
-      const thisAddon = payload.entities.addons[key];
+      const thisAddon = {
+        ...payload.entities.addons[key],
+        isRestartRequired: false,
+      };
       if (thisAddon.theme_data) {
         newState[key] = {
           ...thisAddon,
```